.changeset/khaki-fans-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+'firebase': minor
+---
+
+Bump main package due to minor bumps in auth and database.
```

Comment:
Will this comment go straight into the release notes?

Reply:
No, I'll filter this out when I copy into the CL.